portfolio/src/main/webapp/index.html

```diff
@@ -23,9 +20,11 @@ <h1>About Me</h1>
           reinforcement learning. In my free time I've been trying to develop an agent to play Kung Fu Master in OpenAI Gym
           to mediocre success. 
       </p>
-      <p>You can see more about my time developing iOS apps <a href="appdev.html">here</a>.</p>
-      <p>You can see more about my adventures hiking around UCSC <a href="hiking.html">here</a>.</p>
-      <p>You can see pictures of my lovely puppies <a href="dogs.html">here</a>.</p>
+      <p>You can see more about my time developing iOS apps <a href="appdev.html" class=hlink>here</a>.</p>
+      <p>You can see more about my adventures hiking around UCSC <a href="hiking.html" class=hlink>here</a>.</p>
+      <p>You can see pictures of my lovely puppies <a href="dogs.html" class=hlink>here</a>.</p>
+      <p>Click <a onclick="addRandomGreeting()" class=hlink>here</a> to get a fun fact about me:</p>
```

Comment:
You could rename addRandomGreeting() to getFunFact() to be more accurate

Reply:
Good idea!